jalee/jalee.md

```diff
@@ -0,0 +1,10 @@
+# Jaeha Lee
+## Profile Pic: 
+## Call me by this name: jalee
+## Cohort: July 2023
+## Email ID: dlwohk777@kookmin.ac.kr
+## LinkedIn: no
```

Comment:
It is a very good idea to create a linkedIn profile! If you explicitly do not want to create one then reply to this comment by saying, "I do not want to create my linkedIn profile"